website/_layouts/home.html

```diff
@@ -21,7 +21,8 @@ <h4>Watch it live on the <a href="https://twitch.tv/therubygalaxy">Ruby Galaxy</
          {% include host_header.html hosts=event.hosts %}
          <h5>{{ event.description }}</h5>
       </div>
-      <div class="column">
+      <!-- For future speaker bios and talk abstracts -->
+      <!-- <div class="column">
```

Comment:
Can we us an `if event.talks.count > 0` here?

Reply:
Ok will do!